src/ResourceManager/Sql/Commands.Sql/Failover Group/Services/AzureSqlFailoverGroupCommunicator.cs

```diff
@@ -0,0 +1,154 @@
+        }
+
+        /// <summary>
+        /// Lists Azure Sql Database Failover Groups
+        /// </summary>
+        public IList<Management.Sql.Models.FailoverGroup> List(string resourceGroupName, string serverName, string clientRequestId)
+        {
+            return GetCurrentSqlClient(clientRequestId).FailoverGroups.List(resourceGroupName, serverName).FailoverGroups;
+        }
+
+        /// <summary>
+        /// Creates or updates an Failover Group
+        /// </summary>
+        public Management.Sql.Models.FailoverGroup CreateOrUpdate(string resourceGroupName, string serverName, string FailoverGroupName, string clientRequestId, FailoverGroupCreateOrUpdateParameters parameters)
+        {
+            var resp = GetCurrentSqlClient(clientRequestId).FailoverGroups.CreateOrUpdate(resourceGroupName, serverName, FailoverGroupName, parameters);
+            return resp.FailoverGroup;
+        }
+
+        /// <summary>
+        /// Deletes an Failover Group
+        /// </summary>
+        public void Remove(string resourceGroupName, string serverName, string FailoverGroupName, string clientRequestId)
+        {
+            GetCurrentSqlClient(clientRequestId).FailoverGroups.Delete(resourceGroupName, serverName, FailoverGroupName);
+        }
+
+        /// <summary>
+        /// Fail over an Failover Group without data loss
+        /// </summary>
+        public void Failover(string resourceGroupName, string serverName, string FailoverGroupName, string clientRequestId)
+        {
+            GetCurrentSqlClient(clientRequestId).FailoverGroups.Failover(resourceGroupName, serverName, FailoverGroupName);
+        }
+
+        /// <summary>
+        /// Fail over an Failover Group with data loss
+        /// </summary>
+        public void ForceFailoverAllowDataLoss(string resourceGroupName, string serverName, string FailoverGroupName, string clientRequestId)
+        {
+            GetCurrentSqlClient(clientRequestId).FailoverGroups.ForceFailoverAllowDataLoss(resourceGroupName, serverName, FailoverGroupName);
+        }
+
+        /// <summary>
+        /// Patch-updates an Failover Group
+        /// </summary>
+        public Management.Sql.Models.FailoverGroup PatchUpdate(string resourceGroupName, string serverName, string FailoverGroupName, string clientRequestId, FailoverGroupPatchUpdateParameters parameters)
+        {
+            var resp = GetCurrentSqlClient(clientRequestId).FailoverGroups.PatchUpdate(resourceGroupName, serverName, FailoverGroupName, parameters);
+            return resp.FailoverGroup;
+        }
+
+        /// <summary>
+        /// Lists the Azure Sql Database in the Elastic Pool
+        /// </summary>
+        public IList<Management.Sql.Models.Database> ListDatabasesOnElasticPool(string resourceGroupName, string serverName, string elasticPoolName, string clientRequestId)
```

Comment:
sorry that was used in a previous implementation but we disabled this feature for this release. I will remove it